object/service.proto

```diff
@@ -129,12 +129,10 @@ message HeadResponse {
 }
 
 message SearchRequest {
-    // Version of search query format
-    uint32 Version                           = 1;
```

Comment:
Protocol version from `RequestMetaHeader` and version of query format are not interchangeable. We can have different search query formats without changing the protocol (request format). 

To avoid confusion I suggest to keep `Version` field but rename it to the `QueryVersion` or `QueryFormat`.

Reply:
done